app/services/ember-cli.js

```diff
@@ -80,7 +80,8 @@ const availableBlueprints = {
 const requiredDependencies = [
   'jquery',
   'ember',
-  'ember-template-compiler'
+  'ember-template-compiler',
+  'ember-data'
```

Comment:
While I think ember-data is awesome (and use it often), I also find it useful for demonstration purposes to be able to use JSBin without Ember Data.  Changing this will make it impossible to create a twiddle without Ember Data.

Perhaps, we can still add here and use a `null` value in `twiddle.json` when we want to explicitly not include it?  Not sure what the best path forward here is....